frontend/src/pages/_shared/questionTypes/MultipleChoice.jsx

```diff
@@ -42,6 +43,7 @@ const MultipleChoice = (props) => {
               <RadioGroup aria-label="gender" name="gender1" value={value}>
                 {data.options.map((option) => (
                   <FormControlLabel
+                    key={v4()}
```

Comment:
De id is in dit geval elke keer random, het idee is dat je tussen renders bij kan houden welk element welk element was, dus ik zou in dit geval hier de ib gebruike die meekomt uit de props.